models/DataObject/ClassDefinition.php

```diff
@@ -742,7 +756,13 @@ public function getDefinitionFile($name = null)
             $name = $this->getName();
         }
 
-        $file = PIMCORE_CLASS_DIRECTORY.'/definition_'.$name.'.php';
+        $customFile = PIMCORE_CUSTOM_CONFIGURATION_DIRECTORY . '/classes/definition_'.$name.'.php';
+        if (is_file($customFile)) {
+            return $customFile;
+        } else {
```

Comment:
- No else needed (if has an return)
- return in line 767 is so not reachable

Reply:
The same applies to the other classes.